Wartości Laboratorium EE v1.1.md

```diff
@@ -0,0 +1,167 @@
+## Wartości Laboratorium EE v1.1
+
+Ponieważ kultura organizacji to proces podlegający ciągłym zmianom, chcielibyśmy
+przedstawić Wam w tym dokumencie propozycje core'owych wartości i zasad
+Laboratorium EE.    
+
+**Jednocześnie zapraszamy do jego aktywnego współtworzenia.**
+
+Plik składa się z dwóch części:
+
+1. **Wprowadzenie do Laboratorium EE**
+2. **CodEE Of Conduct**
+
+## Wprowadzenie do Laboratorium EE
+**Laboratorium EE jest firmą technologiczną o profilu badawczo-rozwojowym.
+Naszą misją jest projektowanie i wdrażanie usług cyfrowych, które rozwiązują
+problemy i wspierają rozwój społeczności, w których działamy.**
```

Comment:
Czy „które rozwiązują problemy” odnosi się do wymienionych dalej społeczności? Z szyku za bardzo to nie wynika.